docs/source/sci_api_tutorial/dipole.rst

```diff
@@ -0,0 +1,154 @@
+==============================================
+Bond Dipole
+==============================================
+
+.. note::
+
+    | **This bond_dipole calculation requires you to have 4 CPUs**
+    | **This bond_dipole calculation requires Multiwfn installed**
+
+.. note::
+
+    Science API is a special concept in EnzyHTP. They stands for those top-layer APIs
+    the are supposed to be used in assembling the `workflow <https://enzyhtp-doc.readthedocs.io/en/latest/sci_api_tutorial/how_to_assemble.html#find-the-science-api-that-directly-gives-what-you-need>`_.
+
+**You will find a detailed tutorial of how to use this Science API.**
+
+Input/Output
+=========================================================================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/dipole_bond.svg
+        :width: 100%
+        :alt: dipole_bond
+
+Input
+-----------------------------------------------------------------------------------------
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    The parameters of the bond_dipole API contain the following option
+
+**Input**: The ``ele_stru``, ``atom_1``, ``atom_2`` and  are required input
+
+    - ``ele_stru``:
+        Electronic structure of the QM (Quantum Mechanics) region containing the target bond.
+        .. admonition:: How to obtain
+            
+            | A ElectronicStrutcure() object can be obtained by these `APIs <obtaining_ele_stru.html>`_.
```

Comment:
Mention this is wavefunction file and what formats it takes. (check mulitwfn's doc)

Reply:
We dont need to mention what format it takes here. We will do it in obtaining_ele_stru.html